124-configurable-health-check-response-status.md

```diff
@@ -0,0 +1,70 @@
+# Configurable Health Check Response Status
+
+Add the ability to configure the HTTP response status code returned by the Kafka Bridge health check endpoints (`/ready` and `/healthy`) to accommodate load balancers that only accept HTTP 200 (OK) responses in health checks.
```

Comment:
I think with latest updates this sentence is stale because we are not taking the ability to configure anymore.